packages/pharos/src/components/table/pharos-table.ts

```diff
@@ -0,0 +1,208 @@
+import { PharosElement } from '../base/pharos-element';
+import ScopedRegistryMixin from '../../utils/mixins/scoped-registry';
+import { html, nothing } from 'lit';
+import { state } from 'lit/decorators.js';
+import type { TemplateResult, CSSResultArray } from 'lit';
+import { property } from 'lit/decorators.js';
+import { tableStyles } from './pharos-table.css';
+import { PharosSelect } from '../select/pharos-select';
+import { PharosPagination } from '../pagination/pharos-pagination';
+
+export type ColumnSpecification = {
+  name: string;
+  field: string;
+};
+
+export type RowData = {
+  [key: string]: string;
```

Comment:
Can the row data values really only be strings? I could definitely imagine wanting to pass in `number`s or `null`s as well (maybe others?).

Reply:
updated it to "unknown", not sure if it is the best idea though....